src/main/resources/org/jenkinsci/plugins/workflow/steps/ToolStep/help-type.html

```diff
@@ -0,0 +1,3 @@
+<div>
+   Select the type from the available built-in tool providers <a target="_blank" href="https://www.jenkins.io/doc/book/managing/tools/">here</a>.
```

Comment:
For better accessibility (https://www.w3.org/TR/UNDERSTANDING-WCAG20/navigation-mechanisms-refs.html)
```suggestion
   Select the type from the available <a target="_blank" href="https://www.jenkins.io/doc/book/managing/tools/">built-in tool providers</a>.
```